src/Api/Model/Shared/Dto/ActivityListDto.php

```diff
@@ -51,7 +51,7 @@ public static function getActivityForProject($projectModel, $filterParams = [])
         $dto = ActivityListDtoEncoder::encodeModel($activityList, $projectModel);
         self::prepareDto($dto, $projectModel);
 
-        return (is_array($dto['entries'])) ? $dto['entries'] : [];
+        return (is_array($dto['entries'])) ? array_values($dto['entries']) : [];
```

Comment:
Note to self: this will likely break the existing Activity feed that we are replacing.  So doing a "fast revert" using caddy routing probably won't work since we have changed the data structure the server is responding with.

Reply:
yeah, if I'm understanding you, you're right we couldn't just tweak the proxy but if we were in a situation where we need to rollback, I'd think we'd just go back a version in the images which would not have this.